### PR TITLE
Refactor Email Controller/View

### PR DESCRIPTION
**Pull request by g-rocket**
This is exactly Jess's PR (#186), but with the bad commit removed and rebased onto master.

While I was doing routes for the admin nav bar it was really confusing to figure out why emails were getting routed to weird places, and I also discovered that the show action was really an index action. Just wanted to do a bit of cleanup for whomever we pass this on to, and take some of the logic out of the view.

Also organized the routes.rb a bit lol

### CODE REVIEW

File: app/controllers/static_pages_controller.rb [hunk 1]
```diff
@@ -45,36 +45,36 @@ def pullable_rooms_number (dorm)
...
-          placement_info = [room_assignment.room.dorm.name, 
-                            room_assignment.room.number, 
+          placement_info = [room_assignment.room.dorm.name,
+                            room_assignment.room.number,
                             room_assignment.assignment_type == "preplaced"]
         end
 
         csv << basic_info + placement_info
       end
 		end
```
Comment (g-rocket) [inline app/controllers/static_pages_controller.rb]: Are we mixing tabs and spaces here?

File: config/routes.rb [hunk 2]
```diff
@@ -1,52 +1,43 @@
...
+  get    '/login',   to: 'sessions#new'
+  delete '/logout',  to: 'sessions#destroy'
 
-  resources :users do
-    collection { post :import }
-  end
-  
-  resources :sessions, only: [:create, :destroy]
+  get 'coming_soon', to: 'static_pages#coming_soon'
+  get '/dormLookup', to: 'static_pages#dormLookup'
```
Comment (g-rocket) [inline config/routes.rb]: Why does this start with a slash?  It doesn't seem like routes are supposed to...  Should we fix that while making all these changes?

Comment (jesswinssinger) [reply config/routes.rb]: Maybe. seems strange to me too

**Comment by jesswinssinger** [comment]
LGTM

**Pull request merged**